data/src/frame.rs

```diff
@@ -388,7 +388,16 @@ impl Frame {
                 let s_linesize = src_linesize.next().unwrap();
                 let data = self.buf.as_mut_slice(i).unwrap();
                 let cc = f_iter.next().unwrap();
-                let rr = src.next().unwrap();
+                let rr = src
```

Comment:
would be better zip() and panic if the source and the destination do not match.

Reply:
Don't source and destination have a different memory location and different values? What should I compare?